### PR TITLE
Add section on how to check update results from code

### PR DESCRIPTION
**Pull request by williewillus**

### CODE REVIEW

File: docs/gettingstarted/autoupdate.md [hunk 1]
```diff
@@ -40,3 +40,9 @@ This is fairly self-explanatory, but some notes:
...
 
 Two concrete examples can be seen here for [Charset](https://gist.githubusercontent.com/Meow-J/fe740e287c2881d3bf2341a62a7ce770/raw/bf829cdefc84344d86d1922e2667778112b845b1/update.json) and [Botania Unofficial](https://gist.githubusercontent.com/Meow-J/1299068c775c2b174632534a18b65fb8/raw/42c578cf2303aa76d8900f5fdc6366122549d2a8/update.json).
+
+Retrieving Update Check Results
+-------------------------------
+
+You can retrieve the results of the Forge Update Checker using `ForgeVersion.getResult(ModContainer)`. The returned object has a field `status` which indicates the status of the version check. Example values: `FAILED` (the version checker couldn't connect to the URL provided), `UP_TO_DATE`, `OUTDATED`, `BETA_OUTDATED`. The status will be `PENDING` if the result requested has not finished yet; in that case, you should try again in a little bit. Otherwise, the returned object will also have the target version and any changelog lines, as specified in `update.json`.
```
Comment (elifoster) [inline docs/gettingstarted/autoupdate.md]: Could you specify what `BETA_OUTDATED` indicates? It's not very obvious like `UP_TO_DATE` and `OUTDATED`.

File: docs/gettingstarted/autoupdate.md [hunk 2]
```diff
@@ -40,3 +40,9 @@ This is fairly self-explanatory, but some notes:
...
+You can obtain your own `ModContainer` to pass to this method using `Loader.instance().activeModContainer()`, or any other mod's `ModContainer` using `Loader.instance().getIndexedModList().get(<modid>)`.
```
Comment (elifoster) [inline docs/gettingstarted/autoupdate.md]: You can also use a value annotated with `@Mod.Instance`.

Comment (tterrag1098) [inline docs/gettingstarted/autoupdate.md]: This seems extremely important to mention, as it's probably the way most will be using.

**Comment by tterrag1098** [comment]
Any plans to update this?

File: docs/gettingstarted/autoupdate.md (hunk 2 above)
Comment (williewillus) [inline docs/gettingstarted/autoupdate.md]: but this gives you the mod object itself right? not the `ModContainer`

Comment (williewillus) [inline docs/gettingstarted/autoupdate.md]: (yes, finally looking at this again xP)

Comment (williewillus) [inline docs/gettingstarted/autoupdate.md]: ^ any response to that comment? You want the modcontainer not the actual mod object.

Comment (williewillus) [inline docs/gettingstarted/autoupdate.md]: bump

**Comment by williewillus** [comment]
Any update on this? I'm not sure what exactly needs updating.

File: docs/gettingstarted/autoupdate.md (hunk 1 above)
Comment (tterrag1098) [reply docs/gettingstarted/autoupdate.md]: Yes, I agree with this. It's not clear what the distinction is.

**Comment by tterrag1098** [comment]
Mainly just the confusion about BETA_OUTDATED

**Pull request closed without merging**